apps/studio/src/server/modules/user/__tests__/user.service.test.ts

```diff
@@ -0,0 +1,47 @@
+import { resetTables } from "tests/integration/helpers/db"
+import { setupUser, setUpWhitelist } from "tests/integration/helpers/seed"
+
+import { isUserDeleted } from "../user.service"
+
+describe("user.service", () => {
+  beforeAll(async () => {
+    await resetTables("User")
+    await setUpWhitelist({ email: "@example.com" })
```

Comment:
do we need this? don't see any logic within `isUserDeleted` that's using the whitelisting table + believe this is handled by `isEmailWhitelisted`?

Reply:
good catch! removed